src/components/search/detailed/SearchError.js

```diff
@@ -0,0 +1,35 @@
+/**
+ *
+ * Display search error.
+ *
+ * @author sriram
+ *
+ */
+
+import React from "react";
+import { useTranslation } from "i18n";
+import DEErrorDialog from "components/utils/error/DEErrorDialog";
+import ErrorTypography from "components/utils/error/ErrorTypography";
+
+export default function SearchError(props) {
```

Comment:
heh, I just added almost the exact same component in #168, but I added a generic util component named `ErrorTypographyWithDialog`.

Reply:
Cool.. I will refactor to use that one then.